view/template/trial.hbs

```diff
@@ -69,6 +69,16 @@
         </div>
         <div class="col-sm-12 col-md-7 col-lg-6 col-xl-6">
             <div class="card">
+                    <input type="hidden" id="trialId" name="trialId" value="{{trial.id}}">
```

Comment:
why?
you prevent default on the form an send a custom ajax request.
The router shouldn't know or care that this has changed.